index.html

```diff
@@ -170,15 +202,18 @@ <h4 class="set-h">Skills</h4>
             </ul>
             <img class="set-img" src="images/ic_skills.png">
         </div>
+
+        <img class="dotImg1" src="images/image_dots_with_teal.svg">
         </section>
         <br>
     
 
         <section class="contact">
             <div class="corner__geometry">
-                <img src="./images/corner_geometry.svg">
+                <img class="corner_geometry" src="./images/corner_geometry.svg">
+                <img class="image-geometry_6" src="./images/image-geometry_6.svg">
             </div>
-
+        <div class="get_started">
             <h2 class="h2">Get Started</h2>
             <p class="contact__text">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna.</p>
             <form action="https://formspree.io/f/xeqdpkgq" method="post" class="contact__form">
```

Comment:
- [ ] Kindly try to follow the design in Figma, please align the content of the contact section in the center of the container, also please add the `background-color` as shown in Figma 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/200125054-b2318f90-17ff-4f0b-93c0-5c2cf6239116.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/200125064-83b17908-107f-4c28-817d-6a3cb6541f72.png)